e2e/integration/ideahubtests.js

```diff
@@ -13,12 +13,12 @@ context( 'Test for Idea Hub', () => {
 
     it( 'Test for post a new idea', () => {
         cy.wait(2000)
-        cy.contains( 'Post a new idea!', { timeout: 5000 } ).click();
-        cy.get( '#title' ).type( 'e2e test automation' );
-        cy.get( '#description' ).type( 'e2e test automation' );
-        cy.get( '[aria-label="Options menu"]' ).click();
+        cy.contains( 'Post a new idea!', { timeout: 5000 } ).click({force:true});
+        cy.get( '#title' ).type( 'e2e test automation' ,{force:true});
+        cy.get( '#description' ).type( 'e2e test automation' ,{force:true});
+        cy.get( '[aria-label="Options menu"]' ).click({force:true});
```

Comment:
@mudit94 If we are using `click({force:true});`at multiple places, Can we create a custom command in Cypress and use that across? [Like this](https://stackoverflow.com/questions/61321918/make-forcetrue-in-click-function-the-default-behavior)

Reply:
Hi @ghanlohar , custom command may be used for scenarios like pagination,login,filter selection. as I am doing that in another repo. custom commands are more helpful for achieving a particular functionality (big 1) that needs to be reused.

Reply:
Ok 👍